atom/browser/atom_resource_dispatcher_host_delegate.cc

```diff
@@ -67,10 +73,16 @@ AtomResourceDispatcherHostDelegate::AtomResourceDispatcherHostDelegate() {
 bool AtomResourceDispatcherHostDelegate::HandleExternalProtocol(
     const GURL& url,
     content::ResourceRequestInfo* info) {
+
+  auto render_frame_host =
```

Comment:
this has to be done on the UI thread so just pass all three ids (render_process_id, render_frame_id and frame_tree_node_id)